src/main/java/com/ayuconpon/coupon/controller/request/IssueCouponRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.ayuconpon.coupon.controller.request;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class IssueCouponRequest {
+
+    @NotNull(message = "쿠폰 발급 요청 아이디가 비어있습니다.")
```

Comment:
```suggestion
    @NotBlank(message = "쿠폰 발급 요청 아이디가 비어있습니다.")
```

whitespace 까지 validation 해주는게 어떨까요?

Reply:
`@NotNull` 어노테이션 대신 `@NotBlank` 어노테이션을 사용하여 검증할 경우 아래의 에러메시지가 발생하였습니다.
`No validator could be found for constraint 'jakarta.validation.constraints.NotBlank' validating type 'java.lang.Long'.`

그 이유는 바인딩 되는 타입이 Long이고 Long 타입은 공백 및 whitespace가 들어갈 수 없기 때문인 것으로 생각합니다.

```java
   @NotNull(message = "쿠폰 발급 요청 아이디가 비어있습니다.")
    private Long couponStockId; // 쿠폰 발급 요청 아이디의 타입은 Long
```
==
코드를 다시 보니 `couponStockId` 변수 명은 수정이 되어야 할 것 같습니다..

Reply:
아 Long타입 이었군요 제가 잠시 String으로 착각했네요. 이견없습니다.